buildenv/jenkins/common/build.groovy

```diff
@@ -283,6 +283,9 @@ def build() {
 
 def archive_sdk() {
     stage('Archive') {
+        variableFile.set_sdk_versions()
+        variableFile.set_sdk_variables()
```

Comment:
These calls are not required here. All variables set by these methods are global thus, they are already initialized because `variableFile.set_sdk_variables()` is called by `build_all()` at L572 before the `archive_sdk()`. 
Also `set_sdk_versions()` is redundant here, it is called by `set_sdk_variables()`.